charts/tractusx-connector/values.yaml

```diff
@@ -273,7 +273,7 @@ controlplane:
   # -- configuration of the [Java Util Logging Facade](https://docs.oracle.com/javase/7/docs/technotes/guides/logging/overview.html)
   logging: |-
     .level=INFO
-    org.eclipse.dataspaceconnector.level=ALL
+    org.eclipse.edc.level=ALL
```

Comment:
I'm not sure about this. Can someone confirm, that this needs to be changed like this? Maybe @paullatzelsperger or @tuncaytunc-zf

Reply:
Looks right to me.

Reply:
"org.eclipse.edc.level=ALL" should be the correct one.